.github/workflows/win-release-candidate.yml

```diff
@@ -96,12 +91,12 @@ jobs:
         CLOUDSDK_PYTHON: ${{ env.pythonLocation }}\python.exe
       run: |
         gsutil rm -rf ${{ env.BUCKET_path }} || true
-        gsutil cp -r dist\installers\ ${{ env.BUCKET_path }}
+        gsutil cp dist\installers\Brim-Setup*.exe* dist\installers\latest.yml ${{ env.BUCKET_path }}
```

Comment:
Much like I noted above for macOS, what's expected here:

1.  The `.exe` installer
2.  The `.exe.blockmap` file (described at [electron-userland/electron-builder#2851](https://github.com/electron-userland/electron-builder/issues/2851) as being involved with "partial/differential updates to your application, to shrink the update payload size")
3.  File `latest.yml` (related to auto-update, described somewhat at [https://www.electron.build/auto-update](https://www.electron.build/auto-update))